Updating Host Risk Score documentation with new enablement process (#2308)
elastic/security-docs: docs/experimental-features/host-risk-score.asciidoc

|==============================================

[discrete]
=== Deploy host risk score

To deploy the host risk score framework in your environment:

. https://github.com/elastic/detection-rules/blob/main/docs/experimental-machine-learning/host-risk-score.md#8-enable-kibana-features[Enable the `riskyHostsEnabled` feature flag].
. In the {security-app}, go to *Dashboards* -> *Overview*, then locate the *Current host risk scores* card in the lower-right corner.
+
[role="screenshot"]
image::images/host-risk-score-enable-dev-tools.png[]

> . Click the *Enable via Dev Tools* button, which will take you to the Dev Tools Console in {kib}, pre-populated with all the scripts required to enable host risk score.

Reviewer: A twist on @nastasha-solomon's edits, using streamlined present tense instead of future:
```suggestion
. Click *Enable via Dev Tools*. This opens the Dev Tools Console in {kib} and populates it with the scripts required to enable the host risk score feature.
```

> . Click the *Play* button on each of the scripts numbered 1 through 11.

Reviewer 2: This would be a good thing to fix for accessibility. Not everyone can tell what green is.

> . Click the *Play* button on each of the scripts numbered 1 through 11.

Reviewer: Or maybe this -- I think it's less wordy to say "do this for each script" instead of "do this for script 1, then do it again for the other scripts." Also, I assume the user needs to run the scripts in order, so we should probably emphasize that:
```suggestion
. Click the green triangle to run each of the scripts, in their numbered order from 1 to 11.
```

+
[role="screenshot"]
image::images/host-risk-score-dev-tools-console.png[]

> NOTE: To enable the host risk score feature, you must have alerts in your environment or the *Enable via Dev Tools* button will be disabled.

Reviewer 3: ```suggestion
NOTE: To enable the host risk score feature, you must have alerts in your environment. If you have none, the *Enable via Dev Tools* button is greyed out. 
```

Reviewer 3: Also, this note seems more suited for the intro to these steps (so line 30). This information is most relevant to users _before_ they begin enabling the framework and feature flag because as it says here, unless they have alerts, they won't even be able to use the **Enable via Dev Tools** button.

> The host risk score feature comes with prebuilt dashboards to help you analyze risky hosts in your environment. To import these dashboards:

Reviewer 3: Might be good to turn this into its own section since it's no longer about setting up the framework and enabling the feature. The section title can be something like "Import host risk score dashboard". I'd also preface the instructions for importing the dashboard with a disclaimer that the feature has to be enabled. Here's an example:

```suggestion
[[import-host-risk-score-dashboard]]
[discrete]
=== Import the host risk score dashboard
After you have set up the framework and enabled the feature flag for the host risk score feature, import prebuilt dashboards to help you analyze risky hosts in your environment:
```

> . Navigate to the *Current host risk scores* card on the *Overview* page.

Reviewer 3: ```suggestion
. Navigate to the *Current host risk scores* card (*Dashboards -> Overview* ).
```

> . Click the *Import dashboard* button.

Reviewer 3: ```suggestion
. Click *Import dashboard*.
```

+
[role="screenshot"]
image::images/host-risk-score-import-dashboard.png[]

[[view-host-risk-score]]
[discrete]
=== View host risk score data

> . Navigate to the *Current host risk scores* card on the *Overview* page.

Reviewer 3: ```suggestion
From the *Current host risk scores* card, click *View dashboard*.
```

> . Click *View dashboard*.

Reviewer 3: ```suggestion
```
